cmd/thanos/compact.go

```diff
@@ -303,6 +303,12 @@ func runCompact(
 			if err := sy.SyncMetas(ctx); err != nil {
 				return errors.Wrap(err, "sync before first pass of downsampling")
 			}
+
+			for _, meta := range sy.Metas() {
+				groupKey := compact.DefaultGroupKey(meta.Thanos)
+				downsampleMetrics.downsamples.WithLabelValues(groupKey)
+				downsampleMetrics.downsamples.WithLabelValues(groupKey)
```

Comment:
```suggestion
				downsampleMetrics.downsampleFailures.WithLabelValues(groupKey)
```

Reply:
Done. Thanks for catching this.

Btw just found another bug here https://github.com/thanos-io/thanos/blob/master/cmd/thanos/downsample.go#L234.